test/util/FrontEndTestHelper.scala

```diff
@@ -123,7 +125,7 @@ trait FrontEndTestHelper extends PlaySpec with MockitoSugar with Injecting with
 
     //Return true on the isAuthorized method
     val logic = DefaultSecurityLogic.INSTANCE
-    logic.setAuthorizationChecker((_: WebContext, _: util.List[UserProfile], _: String, _: util.Map[String, Authorizer[_ <: UserProfile]]) => true)
+    logic.setAuthorizationChecker((_: WebContext, _: util.List[UserProfile], _: String, _: util.Map[String, Authorizer[_ <: UserProfile]], _: util.List[Client[_$1]] forSome {type _$1 <: Credentials}) => true)
     testConfig.setSecurityLogic(logic)
```

Comment:
I don't what or how this type works. I copied it off the signature for the method I'm overriding